tests/cmd/e2e/main.go

```diff
@@ -193,6 +201,22 @@ func main() {
 		}
 	}
 
+	// update configuration on the fly
+	for _, clusterInfo := range clusterInfos {
```

Comment:
Need to add this check to e2e?

Reply:
This is a verification of the "ConfigMap Rollout" functionality

Reply:
Sorry, my fault. I mean to add this functionality to `stability` tests.

Reply:
I'm working on the stability test case in another PR now, is that OK or I should merge them together?

Reply:
#486 tracks this

Reply:
another pr is more reasonable